packages/inter-protocol/src/psm/psm.js

```diff
@@ -149,7 +149,7 @@ export const start = async (zcf, privateArgs, baggage) => {
   /**
    * @param {ZCFSeat} seat
    * @param {Amount<'nat'>} given
-   * @param {Amount<'nat'>} [wanted]
+   * @param {Amount<'nat'>} [wanted] defaults to maximum anchor (given exchange rate minus fees)
```

Comment:
Just curious, when would you want anything less than the maximum?

Reply:
not sure myself. A case I can imagine is that if you want a certain amount and you don't know the fees you give a surplus and rely on the contract to only take what's necessary for the want.